.docker.mk

```diff
@@ -64,11 +64,12 @@ docker-%-logs:
 docker-build-debug: docker/debug/Dockerfile.debug
 	@${DOCKERBUILD} -t networkservicemesh/debug -f docker/debug/Dockerfile.debug .
 
-.PHONY: docker-debug-%
-docker-debug-%: docker-build-debug docker-%-kill
-	@${DOCKERBUILD} -t networkservicemesh/$*-debug -f build/Dockerfile.$*-debug .
-	@docker run -d --privileged -p 127.0.0.1:40000:40000 -v "/var/lib/networkservicemesh:/var/lib/networkservicemesh" networkservicemesh/$*-debug > /tmp/container.$*
-	@docker container ls | grep $$(cat /tmp/container.$*| cut -c1-12) > /dev/null && xargs docker logs < /tmp/container.$*
+docker-debug-container: docker-build-debug
+	@docker run --privileged -ti  -p 40000-40100:40000-40100/tcp -v "/var/lib/networkservicemesh:/var/lib/networkservicemesh"  -v $$(pwd | rev | cut -d'/' -f4- | rev):/go/src networkservicemesh/debug /go/src/github.com/ligato/networkservicemesh/scripts/debug_env.sh
+
+.PHONY: docker-debug-run-%
+docker-debug-%:
```

Comment:
We've been following a pattern of:

${context}-${object}-${action}

like
docker-nsmd-build

I'm open to discussing that pattern, but we should probably follow *a* pattern consistently :)

Note: the reason I chose the pattern was because  I found myself doing a sequence of things like:

docker-nsmd-build
docker-nsmd-run
etc
And it was easy to change the action on the end, rather in the middle as I would for

docker-run-nsmd

Reply:
Will update, use one schema is good. Thanks.